tardis/tests/tests_slow/conftest.py

```diff
@@ -39,6 +40,25 @@ def pytest_unconfigure(config):
     shutil.rmtree(config.option.tempdir)
 
 
+@pytest.mark.hookwrapper
+def pytest_runtest_makereport(item, call):
+    # execute all other hooks to obtain the report object
+    outcome = yield
+    report = outcome.get_result()
+
+    if report.when == "call":
+        plot_obj = getattr(item, "plot_obj", None)
```

Comment:
I think we can use `item.funcargs['plot_object']`
Then we don't have to setattr in the fixture